payments/invoice/after-payment.md

```diff
@@ -229,38 +174,54 @@ Content-Type: application/json
 
 {
     "payment": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c",
-    "cancellations": [{
+    "captures": [{
+        "itemDescriptions": {
+            "id": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/transactions/5adc265f-f87f-4313-577e-08d3dca1a26c/itemdescriptions"
+        },
+        "invoiceCopy": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/captures/5adc265f-f87f-4313-577e-08d3dca1a26c/invoicecopy",
         "transaction": {
             "id": "/psp/invoice/payments/5adc265f-f87f-4313-577e-08d3dca1a26c/transactions/5adc265f-f87f-4313-577e-08d3dca1a26c",
             "created": "2016-09-14T01:01:01.01Z",
             "updated": "2016-09-14T01:01:01.03Z",
-            "type": "Cancellation",
-            "state": "Failed",
+            "type": "Capture",
+            "state": "Initialized|Completed|Failed",
```

Comment:
Please don’t use pipes to separate values. Take one value and describe the possible variations and options in the below property table instead. The code examples should be as close to valid requests as possible.